devlib/target.py

```diff
@@ -1359,9 +1361,9 @@ def broadcast_media_mounted(self, dirpath, as_root=False):
         """
```

Comment:
Since this method is being quoted it is probably a good idea to quote the file path in the above `broadcast_media_scan_file` as well.